tpl/fmt/fmt.go

```diff
@@ -41,3 +42,8 @@ func (ns *Namespace) Printf(format string, a ...interface{}) string {
 func (ns *Namespace) Println(a ...interface{}) string {
 	return _fmt.Sprintln(a...)
 }
+
+func (ns *Namespace) Errorf(format string, a ...interface{}) string {
+	jww.ERROR.Printf(format, a...)
```

Comment:
What I suggest you do is:

* Create a Distinct error  https://github.com/gohugoio/hugo/blob/master/helpers/general.go#L304 and put it in Namespace

This will remove the spamming effect and make your test more robust.